### ISSUE TITLE
test failure: t\local\back.t line 53 [rt.cpan.org #106677]

### ISSUE THREAD

**Issue by oalders** [libwww-perl/WWW-Mechanize#88]
Migrated from [rt.cpan.org#106677](https://rt.cpan.org/Ticket/Display.html?id=106677) (status was 'new')

Requestors:
* ether@cpan.org

From ether@cpan.org on 2015-08-26 16:29:28:
```
Via an email I received:

I wanted to install WWW::Mechanize on my new laptop and desktop. But I always got an error message ât\local\back.tâ¦â¦â¦â¦1/47 Error getting http://localhost:51288/:Canât connect to  localhost:51288 at t\local\back.t line 53â . I tried many times and still cannot pass from this line. I actually installed this module at my old laptop and it worked perfect. But it was very long time ago and must be an old version of this module. Do you know how to solve this problem? Maybe it is a bug in this new 1.75 version? Thank you very much!!!
```

**Comment by bouriche021** [comment]
hello frand haw to fix thes erorre  

t/local/back.t ........................... 1/47 Error GETing http://127.0.0.1:38639/: Can't connect to 127.0.0.1:38639 (Connection refused) at t/local/back.t line 53.

**Comment by snehavarg** [comment]
Getting same error 

Mac OS :
t/local/back.t ........................... 1/47 Error GETing http://[::1]:49718/: Can't connect to ::1:49718 (Bad hostname) at t/local/back.t line 53

**Comment by oalders** [comment]
@sherienv do you get this error if you install HTTP::Daemon 6.10? https://metacpan.org/pod/HTTP::Daemon

**Comment by robintwiggs** [comment]
Yes, the issue occurs in HTTP::Daemon version 6.1.0:

...
Checking if you have HTTP::Daemon 6.05 ... Yes (6.10)
...
Error GETing http://[::1]:35205/: Can't connect to ::1:35205 (Bad hostname) at t/local/back.t line 53.

**Comment by oalders** [comment]
I just released `WWW::Mechanize` 1.99. Is that any better?

**Comment by skaji** [comment]
To connect servers via IPv6, we need  IO::Socket::IP and Net::HTTP 6.07+.

IO::Socket::IP must be available because we have added requirement for HTTP::Daemon 6.10.

I would suggest to bump LWP::UserAgent requirement from 5.827 to the latest 6.45.
(LWP::UserAgent  6.45 has requirement for Net::HTTP 6.18)
